src/System Application/App/Edit in Excel/src/EditInExcel.Codeunit.al

```diff
@@ -14,6 +15,42 @@ codeunit 1481 "Edit in Excel"
 {
     Access = Public;
 
+    procedure CreateMetadataWebRequest()
+    var
+        EnvironmentInfo: Codeunit "Environment Information";
+        Token: Text;
+    begin
+        Token := 'herpderp';
+
+
+    end;
+
+    procedure IsMetadataGeneratedForWebService(): Boolean
+    var
+        HttpClient: HttpClient;
+        HttpResponseMessage: HttpResponseMessage;
+        MetadataUrl: Text;
+        EntitySetXml: Text;
+        Document: DotNet XmlDocument;
+        NodeList: DotNet XmlNodeList;
+        NameSpaceManager: DotNet XmlNamespaceManager;
+        EntitySetName: Text;
+    begin
+        MetadataUrl := 'https://metadatastoragexyz.blob.core.windows.net/metadata/metadata.xml?sp=r&st=2024-08-02T09:38:22Z&se=2024-08-02T17:38:22Z&spr=https&sv=2022-11-02&sr=b&sig=eP4pvx1m1OiiGVhYVD4haMIYvSgOAWIUysk3mD7h8Ds%3D';
+        EntitySetName := 'Company';
+        if HttpClient.Get(MetadataUrl, HttpResponseMessage) then
```

Comment:
Do you need any authentication?
![image](https://github.com/user-attachments/assets/088557a9-94a0-4f70-82dc-437a25bb5fb8)